.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
```

Comment:
This is a public repo! we can't have a CI config, can we?

Reply:
all ci-orbs are public and there are no secrets in that orb

Reply:
but doesn't a push from public users create a build in out pipeline?

Reply:
yes ofc, that is how open source works 🤔